static/app/views/performance/newTraceDetails/traceDrawer/details/span/eapSections/description.tsx

```diff
@@ -63,6 +64,10 @@ export function SpanDescription({
   organization: Organization;
   project: Project | undefined;
 }) {
+  const {data: event} = useEventDetails({
+    eventId: node.event?.eventID,
+    projectSlug: project?.slug,
+  });
```

Comment:
This seems to be an EAP section, shouldn't this also construct a synthetic event, rather than fetching the information from event details?

Reply:
Ugh, honestly i didn't want to change how this works, it used to fetch the entire event before, and now it continues to fetch the entire event. @Abdkhan14 Is this fine?